example/src/example_echo_handler.erl

```diff
@@ -0,0 +1,37 @@
+-module(example_echo_handler).
+
+-include_lib("mixer/include/mixer.hrl").
+-mixin([
+        {example_default,
+         [
+          init/3,
+          rest_init/2,
+          content_types_accepted/2,
+          content_types_provided/2,
+          resource_exists/2
+         ]}
+       ]).
+
+-export(
+  [
+   allowed_methods/2,
+   handle_put/2,
+   handle_get/2
+  ]).
+
+%% cowboy
+allowed_methods(Req, State) ->
+  {[<<"GET">>, <<"PUT">>], Req, State}.
+
+%% internal
+handle_get(Req, State) ->
+  Echo = application:get_env(example, echo, ""),
+  Body = [<<"You Get a echo! ">> , Echo ],
+  {Body, Req, State}.
+
+handle_put(Req, State) ->
+  {Echo, Req1} = cowboy_req:binding(echo, Req, ""),
```

Comment:
Remove the binding… or rather… use it as the key for get_env/set_env and return 404 if undefined